templates/show-task.html

```diff
@@ -130,8 +96,12 @@ <h5 class="modal-title" id="add_articles_modal_title"></h5>
                                                      value="" class="article-name"><br/>
             <strong>Description:</strong> <input id="article_description" type="text"
                                                  value="" class="article-description"><br/>
-            <strong>Effort:</strong> <input id="article_effort" type="number" min="0" max="100"
-                                                value="0" class="article-effort"><br/>
+            <strong>Effort:</strong>
```

Comment:
Let's not do it yet (maybe not until we're asked to), but just a thought: I expect that at some point effort will be editable too, either by all users or by the owner of the worklist.